script.js

```diff
@@ -1,4 +1,91 @@
+var gameMode = "roll dice";
+var playerNo = 1;
+
+// get dice roll
+var diceRoll = function () {
+  return Math.ceil(Math.random() * 6);
+};
+
+// create arrays with respective user dice rolls
+var user1 = [diceRoll(), diceRoll()];
+var user2 = [diceRoll(), diceRoll()];
+var userResult = [];
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = " ";
+  var comNum = user1[0] * 10 + user1[1];
+  var comNumRev = user1[1] * 10 + user1[0];
+  var comNum2 = user2[0] * 10 + user2[1];
+  var comNumRev2 = user2[1] * 10 + user2[0];
```

Comment:
Nice way of reversing the numbers! Most would have converted it to a string then back to a number but this works nicely!